src/run_assemble.py

```diff
@@ -30,6 +30,7 @@ def main():
         const=logging.DEBUG,
         dest="logging_level",
     )
+    parser.add_argument("-p", "--path", help="Add the base URL location", type=str)
```

Comment:
No this is actually the path before folder and the component name. 
For example it would be `https://ci.opensearch.org/ci/dbc/bundle-build-opensearch/1.2/608/linux/x64` in the description.

Reply:
+1